Reduce lib size

maksim-tolo opened the pull request:
Hi there!

This PR introduces some changes in build steps and significantly reduces the total size of the library.

Please look at [this](https://github.com/ensdomains/crypto-addr-serialize/pull/3) PR first.

These improvements have been introduced in the PR:
- Added `microbundle` dev dependency. This bundler helps to build the library for different types of modules. This is especially useful for modern applications that use `es6` module system with enabled `tree shaking` feature in their bundlers.
- Replaced `bigi` with `big-integer` dependency (because we already have `big-integer` dependency in another package).
- Replaced heavy `ripemd160` dependency with lightweight `noble-ripemd160`.

Dependencies before the PR:
![image](https://user-images.githubusercontent.com/11413965/76170656-0deeda00-6195-11ea-86eb-5ac3f576d09f.png)

Dependencies after the PR:
![image](https://user-images.githubusercontent.com/11413965/76170600-7a1d0e00-6194-11ea-9538-0f3febb053d1.png)

Arachnid commented:
Thanks! Can you please update this and the dependent PR for the current versions of master?

Arachnid commented:
Also, if you're pulling in code from other libraries, please be sure to reference the source and the license it's under.

maksim-tolo commented:
Seems that https://github.com/ensdomains/address-encoder/pull/37 addressed some issues that have been solved by my PR. I've updated my PRs, but the build is failed because I moved `blakejs` to `crypto-addr-serialize` dependencies. After the merge of the dependent PR, everything should work fine.
